src/Microsoft.Health.Fhir.Core/Messages/Get/GetOperationVersionsResponse.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.Core.Messages.Get
+{
+    public class GetOperationVersionsResponse
+    {
+        public GetOperationVersionsResponse(List<string> supportedVersions, string defaultVersion)
```

Comment:
Use `IList<string>` here

Reply:
Same as above, I run into XML serialization issues if I make this fix.